.readthedocs.yaml

```diff
@@ -1,8 +1,10 @@
 version: 2
-formats: all
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.8"
 python:
-    version: 3
-    install:
+   install:
```

Comment:
You dedented this by a space.  I'd prefer it if the line were kept at a four-space indent.

Reply:
```suggestion
    install:
```

Reply:
@jwodder yarik already fixed this in fb2dc42207077109373601ab75e00f709e089c60

Reply:
@TheChymera That happened after I commented.